.storybook/webpack.config.js

```diff
@@ -21,6 +22,9 @@ const sourceFolders = [
 ];
 
 module.exports = (storybookBaseConfig, configType) => {
+  storybookBaseConfig.plugins.push[
```

Comment:
This looks very odd. You sure `push[1]`?

Reply:
What do you mean `push[1]`? Seems like the same here

https://github.com/storybooks/storybook/issues/2115

Reply:
Do we invoke a function here or dereference an array?

Reply:
Sorry 🤦‍♂️